src/modules/infobox/components/FeatureList.vue

```diff
@@ -103,7 +103,7 @@ function loadMoreResultForLayer(layerId) {
 @import '@/scss/variables.module';
 .feature-list {
     display: grid;
-    grid-template-columns: repeat(auto-fill, minmax($overlay-width, 1fr));
+    grid-template-columns: repeat(auto-fit, minmax($overlay-width, 1fr));
```

Comment:
Ok I see that this grid template is required when displayed in the infobox, but then we might simply add this grid template on the infobox or when the width of the feature-list is above 340px

Reply:
That would make sense to move that to the Infobox (it's meant to fix the use of space here anyway) if this is what caused the scrollbar issue on crhome

Reply:
I ended up activating this "grid" layout only outside of the "fluid" mode, as I couldn't add this CSS only for the InfoboxModule without changing drastically how the HTML was built (or making InfoboxModule aware of the innards of the FeatureList component, which I would like to avoid)

Should be a "good-enough" solution